src/Tizen.NUI/src/public/AccessibilityManager.cs

```diff
@@ -503,412 +506,665 @@ private void OnFocusOvershot(IntPtr currentFocusedView, AccessibilityManager.Foc
+        /// </summary>
+        /// <param name="view"> The view to be set with</param>
+        /// <param name="type"> The attribute type the text to be set with</param>
+        /// <param name="text"> The text for the view's accessibility information</param>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public void SetAccessibilityAttribute(View view, AccessibilityManager.AccessibilityAttribute type, string text)
         {
             Interop.AccessibilityManage.AccessibilityManager_SetAccessibilityAttribute(swigCPtr, View.getCPtr(view), (int)type, text);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Gets the text of the specified view's accessibility attribute.
+        /// </summary>
+        /// <param name="view"> The view to be queried</param>
+        /// <param name="type"> The attribute type to be queried</param>
+        /// <returns> The text of the view's accessibility information </returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public string GetAccessibilityAttribute(View view, AccessibilityManager.AccessibilityAttribute type)
         {
             string ret = Interop.AccessibilityManage.AccessibilityManager_GetAccessibilityAttribute(swigCPtr, View.getCPtr(view), (int)type);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
+        /// <summary>
+        /// Sets the focus order of the view.
+        /// The focus order of each view in the focus chain is unique.
+        /// If there is another view assigned with the same focus order already, the new view will be inserted to the focus chain with that focus order,
+        /// and the focus order of the original view and all the views followed in the focus chain will be increased accordingly.
+        /// If the focus order assigned to the view is 0, it means that view's focus order is undefined
+        /// (e.g. the view has a  description but with no focus order being set yet) and therefore that view is not focusable.
+        /// </summary>
+        /// <param name="view"> the view to be set with</param>
+        /// <param name="order"> the focus order to be set with</param>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public void SetFocusOrder(View view, uint order)
```

Comment:
looks like all the APIs are very similar to FocusManager's APIs. could you explain why?

Reply:
When Accessibility is activated, focus is required as well. Of course, it does not use FocusManager internally, and it has its own focus implementation. The reason for the same API names is to be for the naming unity in DALi, which is C# binding.